.eslintrc

```diff
@@ -0,0 +1,21 @@
+{
+  "parser": "babel-eslint",
+  "extends": ["prettier", "react"],
+  "eol-last": ["error", "always"],
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "plugins": [
+    "react-hooks"
+  ],
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "rules": {
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn"
```

Comment:
`eol-last: ["error", "always"]` is a rule, it should be listed under a `rules` field